src/Dofs/ConstraintHandler.jl

```diff
@@ -855,8 +855,30 @@ end
 
 
 #Function for adding constraint when using multiple celltypes
-function add!(ch::ConstraintHandler, fh::FieldHandler, dbc::Dirichlet)
-    _check_cellset_dirichlet(ch.dh.grid, fh.cellset, dbc.faces)
+function add!(ch::ConstraintHandler{<:MixedDofHandler}, dbc::Dirichlet)
+    dbc_added = false
+    for fh in ch.dh.fieldhandlers
+        if overlaps(fh, dbc)
+            # If dbc have dofs not in fh, then these will be removed from dbc, hence deepcopy
+            # In this case, add! will warn, unless warn_check_cellset=false
+            add!(ch, fh, deepcopy(dbc), warn_check_cellset=false)
```

Comment:
I believe I asked @kimauth this some time -- but why is there a need to copy at all? See also https://github.com/Ferrite-FEM/Ferrite.jl/blob/70f69dd304ee58964d2d31d179d7f15babf36019/src/Dofs/ConstraintHandler.jl#L43 which is what I might have asked about.

Reply:
The deepcopy is required due to https://github.com/Ferrite-FEM/Ferrite.jl/blob/70f69dd304ee58964d2d31d179d7f15babf36019/src/Dofs/ConstraintHandler.jl#L318

Reply:
But perhaps it would be better to rather recreate the entire Dirichlet like in your link, but perhaps here: https://github.com/Ferrite-FEM/Ferrite.jl/blob/70f69dd304ee58964d2d31d179d7f15babf36019/src/Dofs/ConstraintHandler.jl#L313 instead, and then getting rid of that copy upon construction too?

Reply:
Yea I don't really understand why this doesn't just skip.